src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -44,6 +44,7 @@ import { MEDIUM_SCREEN_WIDTH, ResponsiveComponents, SMALL_SCREEN_WIDTH } from '.
 import { FacetColumnAutoLayoutAdjustment } from './FacetColumnAutoLayoutAdjustment';
 import { FacetValueStateHandler } from './FacetValueStateHandler';
 import RelevanceInspectorModule = require('../RelevanceInspector/RelevanceInspector');
+import { AriaLive } from '../../utils/AriaLive';
```

Comment:
It's a tough one, but I think I'd put that "component" under `src/ui/` instead of `utils` , because even though it does not render something "visual", it is still dealing with HTML/JS events etc..